### PR TITLE
feat: new landing page

### PR DESCRIPTION
**Pull request by 2fd**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,7 +4,9 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1,minimal-ui" />
     <link rel="shortcut icon" type="image/x-icon" href="https://decentraland.org/images/icons/favicon-32x32.png" />
-    <link rel="preload" href="https://decentraland.org/images/background@blur.jpg" as="image" />
+    <link rel="preload" href="./images/background-v2@blur.jpg" as="image" />
```
Comment (menduz) [inline public/index.html:7]: Please use %PUBLIC_URL% everywhere in the HTML to make it work with the static CDN

```suggestion
    <link rel="preload" href="%PUBLIC_URL%/images/background-v2@blur.jpg" as="image" />
```

**Review by menduz — CHANGES_REQUESTED** [review]

File: src/components/auth/LoginContainer.css [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
+  display: flex;
+  align-items: center;
+  height: 100vh;
+  min-height: 790px;
+  background-size: cover;
+  background-color: transparent;
+  background-position: bottom center;
+  background-repeat: no-repeat;
+  background-origin: border-box;
+  background-image: url('../../images/background-v2@1x.jpg');
```
[the remark this answers is not in the document]
Comment (2fd) [reply src/components/auth/LoginContainer.css:16]: I tested it, it works as expected
Screen Shot 2021-10-14 at 13 49 45

**Comment by 2fd** [comment]
I see it good on mi firefox, anyone else ?

Screen Shot 2021-10-14 at 14 57 35

**Comment by menduz** [comment]
@cazala knows the issue with wide fonts, it already happened in the past

File: src/components/App.tsx [hunk 3]
```diff
@@ -25,14 +36,18 @@ const App: React.FC<AppProps> = (props) => {
     return <ErrorContainer />
   }
 
-  if (props.rendererVisible) {
-    return <React.Fragment />
+  if (props.sessionReady) {
```
Comment (menduz) [inline src/components/App.tsx:39]: This _also_ needs to disappear once the renderer is visible.
```suggestion
  // We had a performance regression in the past where the loading GIF behind the renderer
  // raised CPU usage levels and reduced the FPS significantly
  if (props.rendererVisible) {
    return <React.Fragment />
  }
  if (props.sessionReady) {
```

Comment (2fd) [reply src/components/App.tsx:39]: It does, but that is control in other place so it can disappear with an animation

**Review by menduz — CHANGES_REQUESTED** [review]

**Comment by olavra** [comment]
I thing we are doubling the risk with the festival so close, having two versions of the front increases chance of failure. Both Romi and I agree the "hidden animated button" was comprehensible enough for everybody. If nobody has an issue with it, please lets go 100% with that and release asap. We need time to react to the analytics before the event.

**Review by menduz — APPROVED** [review]
Left my approval there, ping me when you want to test it after merge

File: src/components/auth/LoginContainer.css [hunk 4]
```diff
@@ -20,97 +38,58 @@
...
 
-  .eth-login-popup .eth-login {
-    min-height: auto;
+  main.LoginContainer > .eth-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    margin: 0 auto;
+    /* height: calc(100% - 128px); */
+    /* margin: 64px auto; */
```
Comment (nachomazzara) [inline src/components/auth/LoginContainer.css:88]: Do we need to keep these comments?

File: src/components/auth/LoginContainer.tsx [hunk 5]
```diff
@@ -1,27 +1,26 @@
...
 import { authenticate } from '../../kernel-loader'
-import BannerContainer from '../banners/BannerContainer'
-import DownloadProgress from '../desktop/DownloadProgress'
+import { EthWalletSelector } from './EthWalletSelector'
+import { LoginGuestItem, LoginWalletItem } from './LoginItemContainer'
 import { isElectron } from '../../integration/desktop'
+import logo from '../../images/logo.png'
+import './LoginContainer.css'
+// import { EthSignAdvice } from './EthSignAdvice'
+// import { EthConnectAdvice } from './EthConnectAdvice'
```
Comment (nachomazzara) [inline src/components/auth/LoginContainer.tsx:15]: same here

File: src/components/auth/LoginContainer.tsx [hunk 6]
```diff
@@ -40,40 +39,54 @@ export interface LoginContainerProps {
...
-        </main>
-        {!isElectron() && <BeginnersGuide /> }
-        {!isElectron() && <BigFooter /> }
-      </div>
-    </React.Fragment>
+    <main className="LoginContainer">
+    {/* {stage === LoginState.CONNECT_ADVICE && <EthConnectAdvice onLogin={onLogin} />} */}
+    {/* {stage === LoginState.SIGN_ADVICE && <EthSignAdvice />} */}
+    <Container>
+        {/* {isElectron() && <DownloadProgress/> } */}
```
Comment (nachomazzara) [inline src/components/auth/LoginContainer.tsx:71]: same here

**Review by nachomazzara — CHANGES_REQUESTED** [review]
LGTM. Just minor comments

File: src/state/reducers.ts [hunk 7]
```diff
@@ -54,8 +54,9 @@ export function sessionReducer(state: SessionState | undefined, action: AnyActio
 }
 
 export function rendererReducer(state: RendererState | undefined, action: AnyAction): RendererState {
-  if (state && action.type === SET_RENDERER_COMPLETE) {
-    return { ...state, complete: action.payload.complete }
+  console.log(action.type, action.payload)
```
Comment (menduz) [inline src/state/reducers.ts:57]:
```suggestion
```

**Pull request merged**